code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1,5 +1,5 @@
 /datum/reagent/blood
-	data = list("viruses"=null,"blood_DNA"=null,"blood_type"=null,"resistances"=null,"trace_chem"=null,"mind"=null,"ckey"=null,"gender"=null,"real_name"=null,"cloneable"=null,"factions"=null,"quirks"=null)
```

Comment:
This can be done modularly.

Reply:
It can, however, considering how ugly this list is, it's probably very likely to get modified upstream (I had half a mind of making that change upstream myself, just haven't gotten to it yet), and I'd rather for any behavior change that comes from that to not be missed on accident from a mirror merged a little too hastily.